lib/presentation/crowdactions/crowdaction_view/commitment_item.dart

```diff
@@ -22,6 +22,8 @@ class CommitmentItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    double width = size.width;
```

Comment:
This code is currently unused. Let's remove it

Reply:
@CaglarKullu Sorry, I didn't see that the width variable was being used. However, we actually don't need a `SizedBox`. Wrapping the `Container` with `Expanded` does the trick because, without `Expanded`, the `Container` inherits the constraints of the parent. In this case, the parent is a `Row`, so the `Container` gets infinite constraints (that is the behavior of Rows). These constraints get passed down to the children, which are the `Text` widgets. That's why the text overflows the `Container` - because its constraints are set to infinity. However, when you wrap the `Container` with `Expanded`, it fills the available space, which in this case would be the constraints of the main container.